components/lightsnapcast/Kconfig.projbuild

```diff
@@ -0,0 +1,18 @@
+menu "Player-Settings"
+config FLIP_MCLK_LEVEL
+            bool "Invert Logic Level MCLK"
+            default false
+
+config FLIP_WORD_SELECT_LEVEL
+            bool "Invert Logic Level WordSelect"
+            default false
+            help
+                Some Dacs can bridge their outputs. Word-Select is used for selecting the played audio channel.
+config FLIP_BCLK_LEVEL
+            bool "Invert Logic Level BCLK"
+            default false
+config DISABLE_APLL_ADJUST
```

Comment:
Please add some description what this is doing, e.g. disable sync implementation!

Reply:
Is done in the next commits. I will push them soon.

Reply:
@DerPicknicker I don't like that you can disable sync implementation. Also I changed your merge request to sample stuffing branch which will include a choice for apll  s sample stuffing. Please remove this setting

Reply:
@CarlosDerSeher  Just remove the setting inside the Menuconfig or also the line in the code where I use it? I guess the second option ;-)